assets/src/css/frontend/give-frontend.scss

```diff
@@ -26,6 +26,9 @@
 @import 'modal';
 @import '_components.donor';
 
+// Give Blocks common styles (used on frontend and in editor)
+@import '../../../../blocks/milestone/common.scss';
```

Comment:
As a note, this is tentative based on where we go with shortcodes and server-side rendering.

Reply:
I thought through this point a lot while working on this scaffolding, and am of course open to your feedback here - but wanna explain my own reasoning behind the decision:

According to the pitch, we plan on introducing Milestones as a shortcode as well - which is why I opted to go with this "common" css approach. This is also the approach currently used by other blocks (because they all double as shortcodes as well).

Additionally, a common refrain from GiveWP users is that we _already_ enqueue too many scripts/styles on each page. Of course, block registration is smart enough to ensure that block specific styles are only enqueued on pages that actually utilize the block, but until we see this specific block's styles grow significantly it will likely be more load efficient to add these styles to the frontend stylesheet Give loads. 

Ultimately, if you'd prefer I moved it, it's a quick change and happy to do so!